### DISCUSSION TITLE
Weather IVM - Artificial intelligence (AI) ChatGPT

### DISCUSSION THREAD

**Discussion by HauHetCo** [community/community#53147] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This project is a Python script that allows you to get the current weather data for a given city using the OpenWeatherMap service API.

https://github.com/StratonPro/Weather-IVM/tree/main/ChatGPT

**Comment by queenofcorgis** [comment]
Hi @StratonPro , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by HauHetCo** [comment]
thank you!
